### DISCUSSION TITLE
Allow the NetworkTransform to update the Component Configuration when the owner changes with Client Authoritative enabled.

### DISCUSSION THREAD

**Discussion by MikeMuorte** [FirstGearGames/FishNet#777] [Feature Request]
Use the **ConfigureComponents** method inside the **OnOwnershipClient** method, which would allow updating the *Rigidbody, Rigidbody2D, and CharacterController* components when a **NetworkTransform** changes owner and the component is enabled as **Client Authoritative**.

**Comment by MikeMuorte** [comment]
Just add **ConfigureComponents();** in **OnOwnershipClient** (line 684), like this:
```
public override void OnOwnershipClient(NetworkConnection prevOwner)
{
     ConfigureComponents();
     _intervalsRemaining = 0;
     ...
}

**Comment by FirstGearGames** [comment]
Added in 4.4.8
